src/ImageSharp.Drawing/Paths/InternalPath.cs

```diff
@@ -203,6 +247,92 @@ public bool PointInPolygon(Vector2 point)
             return oddNodes;
         }
 
+        private static bool BoundingBoxesIntersect(Vector2 line1Start, Vector2 line1End, Vector2 line2Start, Vector2 line2End)
```

Comment:
yeah my bad, i'm trying to catch myself using `var` but I do miss them.

We should think about trying to find an analyzer that can catch them.